app/src/main/java/com/maciejmalak/githubstatistics/activities/UserDetailedView.java

```diff
@@ -2,34 +2,37 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.bumptech.glide.Glide;
 import com.maciejmalak.githubstatistics.R;
-import com.maciejmalak.githubstatistics.model.GithubStatisticModel;
+import com.maciejmalak.githubstatistics.model.Owner;
 
-import butterknife.Bind;
+import butterknife.BindView;
 import butterknife.ButterKnife;
 
 public class UserDetailedView extends AppCompatActivity {
-
-    final GithubStatisticModel mModel = GithubStatisticModel.getInstance();
-
-    @Bind(R.id.detailed_user_name) protected TextView mTvUserName;
-    @Bind(R.id.detailed_nbr_repos) protected TextView mTvNbrRepos;
+    @BindView(R.id.detailed_user_name) protected TextView mUserName;
+    @BindView(R.id.detailed_avatar) protected ImageView mUserAvatar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_user_detailed_view);
 
         ButterKnife.bind(this);
-
-        loadDetailedUserData(getIntent().getExtras().getString(UserDetailedView.class.getName()));
+        /*TODO How to avoid passing entire object from activity to another*/
+        final Owner user = (Owner) getIntent().getExtras().getSerializable(UserDetailedView.class.getName());
+        if (user != null) loadDetailedUserData(user);
+//        else TODO
```

Comment:
Apply presenter to this class in future. Owner should be passed to presenter, callbacks to view should receive already prepared data from presenter (e.g. login text).